src/payment/strategies/googlepay/googlepay-braintree-initializer.ts

```diff
@@ -79,7 +79,7 @@ export default class GooglePayBraintreeInitializer implements GooglePayInitializ
             transactionInfo: {
                 currencyCode: checkout.cart.currency.code,
                 totalPriceStatus: 'FINAL',
-                totalPrice: checkout.grandTotal.toString(),
+                totalPrice: checkout.grandTotal.toFixed(2),
```

Comment:
Can you do `round(checkout.grandTotal, 2).toFixed(2)`? Because some currencies have more than 2 decimal places, so we want to round the number first before padding the number with 0.